themes-default/slim/src/components/display-show.vue

```diff
@@ -265,25 +285,20 @@ export default {
             $.tablesorter.columnSelector.attachTo($('#showTable, #animeTable'), '#popover-target');
         });
 
-        // Moved and rewritten this from displayShow. This changes the button when clicked for collapsing/expanding the
-        // Season to Show Episodes or Hide Episodes.
-        $('.collapse.toggle').on('hide.bs.collapse', function() {
+        // Changes the button when clicked for collapsing/expanding the season to show/hide episodes
+        document.addEventListener('hide.bs.collapse', function() {
             const reg = /collapseSeason-(\d+)/g;
             const result = reg.exec(this.id);
             $('#showseason-' + result[1]).text('Show Episodes');
             $('#season-' + result[1] + '-cols').addClass('shadow');
         });
-        $('.collapse.toggle').on('show.bs.collapse', function() {
+        document.addEventListener('show.bs.collapse', function() {
```

Comment:
There's a good chance this is broken.

Reply:
I suggest:
```js
document.querySelectorAll('.collapse.toggle').forEach(element => {
    element.addEventListener('hide.bs.collapse', function() {
        // on hide
    });
    element.addEventListener('show.bs.collapse', function() {
        // on show
    });
));
```

And to test this you need to disable `Show all seasons` in General Config -> Interface.